components/eam/cime_config/config_component.xml

```diff
@@ -106,47 +88,36 @@
     <valid_values></valid_values>
     <default_value>UNSET</default_value>
     <values>
-      <value compset="RCP2_EAM"        >2006-2100_cam5_rcp26</value>
-      <value compset="RCP4_EAM"        >2006-2100_cam5_rcp45</value>
-      <value compset="RCP6_EAM"        >2006-2100_cam5_rcp60</value>
-      <value compset="RCP8_EAM"        >2006-2100_cam5_rcp85</value>
-      <value compset="2000(?:SOI)?_EAM%MOZM"   >2000_cam5_trop_moz_mam3</value>
-      <value compset="1850(?:SOI)?_EAM%PM"     >1850_cam5_pm</value>
-      <value compset="2000(?:SOI)?_EAM.*PLMOD" >2000_cam5_plmod</value>
-      <value compset="1850(?:SOI)?_EAM.*CMIP6"  >1850_eam_CMIP6</value>
+      <value compset="RCP2_EAM"                         >2006-2100_cam5_rcp26</value>
+      <value compset="RCP4_EAM"                         >2006-2100_cam5_rcp45</value>
+      <value compset="RCP6_EAM"                         >2006-2100_cam5_rcp60</value>
+      <value compset="RCP8_EAM"                         >2006-2100_cam5_rcp85</value>
+      <value compset="1850(?:SOI)?_EAM.*CMIP6"          >1850_eam_CMIP6</value>
       <value compset="1850(?:SOI)?_EAM.*CMIP6-1pctCO2"  >1850_eam_CMIP6-1pctCO2</value>
-      <value compset="1950(?:SOI)?_EAM.*CMIP6"  >1950_eam_CMIP6</value>
+      <value compset="1950(?:SOI)?_EAM.*CMIP6"          >1950_eam_CMIP6</value>
       <value compset="1850(?:SOI)?_EAM.*CMIP6.*_BGC%*"  >1850_eam_CMIP6_bgc</value>
-      <value compset="2010(?:SOI)?_EAM.*CMIP6"  >2010_eam_CMIP6</value>
-      <value compset="1850(?:SOI)?_EAM.*AR5sf"   >1850_E3SMv1_superfast_ar5-emis</value>
-      <value compset="1850S_EAM.*AR5sf"  >1850S_E3SMv1_superfast_ar5-emis</value>
-      <value compset="20TR(?:SOI)?_EAM.*CMIP6"  >20TR_eam_CMIP6</value>
-      <value compset="20TR(?:SOI)?_EAM.*CMIP6.*_BGC*"  >20TR_eam_CMIP6_bgc</value>
-      <value compset="SSP585(?:SOI)?_EAM.*CMIP6">SSP585_cam5_CMIP6</value>
-      <value compset="SSP585(?:SOI)?_EAM.*CMIP6.*_BGC%*">SSP585_cam5_CMIP6_bgc</value>
-      <value compset="20TR(?:SOI)?_EAM.*AR5sf" >20TR_E3SMv1_superfast_ar5-emis</value>
-      <value compset="20TRS_EAM.*AR5sf">20TRS_E3SMv1_superfast_ar5-emis</value>
-      <value compset="1850(?:SOI)?_EAM%WCCM"   >waccm_1850_cam5</value>
-      <value compset="2000(?:SOI)?_EAM%WCCM"   >waccm_2000_cam5</value>
-      <value compset="_EAM%SMA3"       >2000_cam5_trop_strat_mam3</value>
-      <value compset="_EAM%SMA7"       >2000_cam5_trop_strat_mam7</value>
-      <value compset="GEOS_EAM%SMA3"   >sd_cam5_trop_strat_mam3</value>
-      <value compset="AR95_EAM"     >scam_arm95</value>
-      <value compset="AR97_EAM"     >scam_arm97</value>
-      <value compset="SCAM_EAM"        >scam_generic</value>
```

Comment:
Need to retain the entry for scam_generic, with the compset name updated accordingly as in config_compsets.xml for F_SCAM

Reply:
ok, I'll restore a generic SCM compset similar to how the F_SCAM compset was configure.